File: 6 kyu Reverse polish notation calculator.py
import operator

def calc1(expr):
    # TODO: Your awesome code here
    op = {'+': operator.add, '-': operator.sub, '*': operator.mul, '/': operator.truediv}
    res = [0]
    for token in expr.split(' '):
        if token in op:
            res.pop()
            res.pop()
            res.append(op[token](op1, op2))
        elif token:
            res.append(float(token))
    return res.pop()

print(calc1("5 1 2 + 4 * + 3 -"))

File: 6 kyu Reverse polish notation calculator.js
function calc(expr)
{
    // TODO: Your awesome code here
    expr = expr.split(' ');
    let res = []
    const op = {
        '+': function (x, y) {return x + y;},
        '-': function (x, y) {return x - y;},
        '*': function (x, y) {return x * y;},
        '/': function (x, y) {return x / y;},
    }

    for (let i = 0; i < expr.length; i++)
    {
        if (isNaN(expr[i]))
        {
            res.push(op[i]( parseFloat(res.pop), parseFloat(res.pop)));
        }
        else
        {
            res.push(expr[i]);
        }
    }
        
            return res;
}
    
console.log(calc("4 2 /"));